kwave/utils/mapgen.py

```diff
@@ -2466,7 +2467,7 @@ def focused_bowl_oneil(radius: float, diameter: float, velocity: float, frequenc
         lateral_position = np.arange(-15e-3, 15e-3 + 1e-4, 1e-4)  # [m]
 
         # evaluate pressure
-        [p_axial, p_lateral] = focused_bowl_oneil(radius, diameter,
+        [p_axial, p_lateral, p_axial_complex] = focused_bowl_oneil(radius, diameter,
```

Comment:
Consider removing square brackets as they are related to Matlab, not Python.

Reply:
done